api/inc/register_gateway.h

```diff
@@ -20,6 +20,8 @@
 #include "api/inc/uvisor_exports.h"
 #include <stdint.h>
 
+#define UVISOR_REGISTER_GATEWAY_MAGIC 0xCDCDCDCD
```

Comment:
@AlessandroA the magic should not contain identical bytes in sequence. Ideally both halves are the same illegal opcode to ensure that normal thumb code will never have this sequence.